code/modules/projectiles/ammunition/boxes.dm

```diff
@@ -52,13 +52,11 @@
 	name = "ammunition packet (.35 Auto practice)"
 	icon_state = "pistol_p"
 	ammo_type = /obj/item/ammo_casing/pistol/practice
-	spawn_blacklisted = TRUE
 
 /obj/item/ammo_magazine/ammobox/pistol/hv
 	name = "ammunition packet (.35 Auto high-velocity)"
 	icon_state = "pistol_hv"
 	ammo_type = /obj/item/ammo_casing/pistol/hv
-	spawn_blacklisted = TRUE
```

Comment:
Hv ammo should probably have a higher rarity than normal ammo, if it already isn't more rare.

Reply:
The HV ammo seems to be much more rarer than other boxes for god what knows reason. Including that they also rarely spawn fully-stocked this barely a problem.